fs/copy.ts

```diff
@@ -161,7 +161,7 @@ function copySymlinkSync(
 ) {
   ensureValidCopySync(src, dest, options);
   const originSrcFilePath = Deno.readLinkSync(src);
```

Comment:
Using `Deno.realPathSync` should return the absolute path of the symlink target. Additional explanation for suggestion in PR #6236:
```suggestion
  const originSrcFilePath = Deno.realPathSync(src);
```